accepted/2021/statics-in-interfaces/README.md

```diff
@@ -667,21 +671,30 @@ The numeric interfaces build upon the base interfaces by defining the core abstr
 ```csharp
 namespace System
 {
-    public interface INumber<TSelf>
+    public interface INumberBase<TSelf>
```

Comment:
Given that one of the prime examples was to support `Complex`, this would likely be something like:
```csharp
public interface IComplexNumber<TSelf, TNumber>
    : IDivisionOperators<TSelf, TSelf, TSelf>,
      IMultiplicativeIdentity<TSelf, TSelf>,
      INumberBase<TSelf>,
      ISpanFormattable<TSelf>,
      IUnaryNegationOperators<TSelf, TSelf>
    where TSelf : IComplexNumber<TSelf, TNumber>
    where TNumber : INumber<TNumber>
{
    public static TSelf ImaginaryOne { get; }
    public static TSelf Infinity { get; }
    public static TSelf Nan { get; }
    public static TSelf One { get; }

    public TNumber Imaginary { get; }
    public TNumber Magnitude { get; }
    public TNumber Phase { get; }
    public TNumber Real { get; }

    public static TNumber Abs(TSelf value);
    public static TSelf Conjugate(TSelf value);

    // Acos, Asin, Atan
    // Cos, Sin, Tan
    // Cosh, Sinh, Tanh
    // Exp, Log, Log10. Pow
    // Reciprocal, Sqrt
    // IsFinite, IsInfinity, IsNaN
}
```

Where `struct Complex : IComplexNumber<Complex, double>`

Reply:
This doesn't cover things like `IAdditionOperators<TSelf, TSelf, TNumber>`

It doesn't cover splitting out functions like `Sin`/`Cos`/`Tan` into some interface that can be shared between `IComplexNumber` and `IFloatingPoint`

Reply:
> `IAdditionOperators<TSelf, TSelf, TNumber>`

Did you mean `IAdditionOperators<TSelf, TNumber, TSelf>`? Perhaps better would be to just define an implicit conversion from `TNumber` to `TSelf`.

Reply:
> Did you mean IAdditionOperators<TSelf, TNumber, TSelf>

Yes. Fixed above.